Cour-de-cassation/label
# Reopen magistrat categorie

## Issue description :
Make `professionnelMagistratGreffier` category annotable to correct NLP annotator error but do not anonymize if it's in `categoriesToOmit`.

## Describe your changes :
Use the `visible` status of categories settings and allow users to annotate element in that category.

## How to test :
- annotate some terms in the `professionnelMagistratGreffier`
- if this category is in `categoriesToOmit`, make sure that in the anonymize view and at export terms are NOT annotated
- if this category is in NOT `categoriesToOmit`, make sure that in the anonymize view and at export terms are annotated

## Checklist before requesting a review

> Désormais la categorie `professionnelMagistratGreffier` pourra être annotée tout en restant non pseudonymisés pour corriger les erreurs d'annotation du moteur. A la demande de la juridiction, les magistrat et les greffiers pourront être pseudonymisés. Dans ce cas l'annotation dans label sera pleine et non soulignée. Par conséquent les annotations des magistrats et greffiers seront disponibles en relecture simple.
